packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -91,6 +91,14 @@ void main() {
       fakeVideoPlayerPlatform = FakeVideoPlayerPlatform();
     });
 
+    test('plugin initialized', () async {
```

Comment:
This test seems to fail on CI.
I think a previous test case may already be triggering the initialization and thus this one doesn't.

I'm not sure if package:test allows controlling the order of test execution. Maybe we can split this test case to it's own file to ensure it's the first (and only) test for the given engine instance?

Reply:
Sounds good. I have no idea how I would do this unfortunately.

Reply:
Create another file named `video_player_initialization_test.dart` with just this test